src/routes/github/webhook/webhook-receiver-post.ts

```
@@ -71,3 +68,11 @@ const createHash = (data: BinaryLike, secret: string): string => {
 		.update(data)
 		.digest("hex")}`;
 };
+
+const getWebhookSecret = async (uuid?: string) => {
+	if (uuid) {
+		const gitHubServerApp = await GitHubServerApp.findForUuid(uuid);
+		return gitHubServerApp && await gitHubServerApp.decrypt("webhookSecret");
```

This should probably throw an error if `GitHubServerApp` isn't found.